github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-presubmits.yaml

```diff
@@ -106,7 +106,55 @@ presubmits:
         resources:
           requests:
             memory: "8Gi"
-
+  - name: check-kubevirt-e2e-k8s-1.18
+    always_run: false
+    optional: true
+    decorate: true
+    skip_report: false
+    extra_refs:
+      - org: kubevirt
+        repo: kubevirt
+        base_ref: master
+    decoration_config:
+      timeout: 7h
+      grace_period: 5m
+    max_concurrency: 1
+    labels:
+      preset-dind-enabled: "true"
+      preset-docker-mirror-proxy: "true"
+      preset-shared-images: "true"
+      preset-bazel-cache: "true"
+      preset-bazel-unnested: "true"
+    spec:
+      nodeSelector:
+        type: bare-metal-external
+      containers:
+        - image: gcr.io/k8s-testimages/kubekins-e2e@sha256:8c80c98d035a0f285ad49b964e5d0b62004844f3340407367841618e80e1503c
+          command:
+            - "/usr/local/bin/runner.sh"
```

Comment:
We can not accept jobs anymore which do their stuff in the command section here. Jobs need a repo where the logic is placed, so that all changes can be evolved together.